src/main/java/com/minecolonies/util/BlockUtils.java

```diff
@@ -109,138 +113,201 @@ public static boolean isWater(IBlockState iBlockState)
     }
 
     /**
-     * Gets an item from a block.
-     * @param block input block.
-     * @return output item.
+     * Mimics pick block.
+     * @param blockState the block and state we are creating an ItemStack for.
+     * @return ItemStack fromt the BlockState.
      */
-    public static Item getItemFromBlock(Block block)
+    public static ItemStack getItemStackFromBlockState(IBlockState blockState)
     {
-        Item item = null;
+        Item item = getItem(blockState);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'item' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)